network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -109,7 +109,11 @@ class Meta:
 
 
 class BlogPage(FoundationMetadataPageMixin, Page):
-    body = StreamField(base_fields)
+    body = StreamField(
+        base_fields + [
+            ('airtable', customblocks.AirTableBlock()),
```

Comment:
what other pages use `base_fields`? Because depending on those, it might make sense to just add this to the full list.

Reply:
Hi Pomax! It looks like there are a number of different pages that use [base_fields](https://github.com/mozilla/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/pagemodels/base_fields.py). However, I did note that some of these pages, such as the blog and article pages, are defining their own list of "base_fields".

For example, here is a link to the article page base fields: https://github.com/mozilla/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/pagemodels/article_fields.py

And then here is a link to where the blog page is defining its own "base_fields".

Reply:
nice, thanks!